packages/material-ui/src/FormControlLabel/FormControlLabel.test.js

```diff
@@ -17,6 +18,10 @@ describe('<FormControlLabel />', () => {
     classes = getClasses(<FormControlLabel label="Pizza" control={<div />} />);
   });
 
+  afterEach(() => {
```

Comment:
afterEach or after? We mix the two for testing-library in the codebase.

Reply:
afterEach seems to be encouraged in the docs but after seems to work fine.

Reply:
Yeah we currently have an implicit afterEach but I want to use afterEach wherever possible so that it's easier to identify which tests need work.